packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -2342,3 +2339,116 @@ describe("DeleteAttributeUseCases", () => {
         });
     });
 });
+
+describe("ThirdPartyRelationshipAttributes", () => {
+    let localAttribute: LocalAttributeDTO;
+    beforeAll(async () => {
+        await cleanupAttributes();
+        localAttribute = await executeFullCreateAndShareRelationshipAttributeFlow(services1, services2, {
+            content: {
+                key: "ThirdPartyKey",
+                confidentiality: RelationshipAttributeConfidentiality.Public,
+                value: {
+                    "@type": "ProprietaryString",
+                    value: "ThirdPartyValue",
+                    title: "ThirdPartyTitle"
+                },
+                isTechnical: true
+            }
+        });
+    });
+
+    test("should share a ThirdPartyRelationshipAttribute that was created by the sharing identity", async () => {
+        const localThirdPartyAttribute = await executeFullShareAndAcceptAttributeRequestFlow(
+            services1,
+            services3,
+            ShareAttributeRequestItem.from({
+                attribute: localAttribute.content,
+                sourceAttributeId: localAttribute.id,
+                thirdPartyAddress: services2.address,
+                mustBeAccepted: true
+            })
+        );
+
+        const services1AttributesResult = (await services1.consumption.attributes.getAttribute({ id: localThirdPartyAttribute.id })).value;
+        const services3AttributesResult = (await services3.consumption.attributes.getAttribute({ id: localThirdPartyAttribute.id })).value;
+
+        expect(services1AttributesResult.shareInfo!.thirdPartyAddress).toStrictEqual(services2.address);
+        expect(services3AttributesResult.shareInfo!.thirdPartyAddress).toStrictEqual(services2.address);
+    });
+
+    test("should share an ThirdPartyRelationshipAttribute that was shared with the sharing identity", async () => {
```

Comment:
"**a** ThirdPartyRelationshipAttribute" instead of "an ThirdPartyRelationshipAttribute".